main.py

```diff
@@ -235,6 +235,8 @@ def __init__(
         self.bitrate = bitrate
         self.bus = None
         self.task = None
+        self.skew_map = {}  # This store the couple [ ID | Estimated CK (Clock Skews) ]

```

Comment:
Missing id-specific maps for mu_T, error and timestamps, but I'll add them later